src/three-components/SmoothControls.ts

```diff
@@ -357,10 +387,19 @@ export class SmoothControls extends EventDispatcher {
     const targetTheta = theta - deltaTheta;
     const targetPhi = phi - deltaPhi;
     const targetRadius = radius + deltaRadius;
-    
+
     return this.setOrbit(targetTheta, targetPhi, targetRadius);
   }
 
+  /**
+   * Move the camera instantly instead of accelerating toward the setOrbit()
+   * parameters.
+   */
+  jumpCamera() {
```

Comment:
The reason I changed it is because it referred to $targetSpherical, which is an implementation detail. We intend to add a configurable camera target, at which point this function would start to sound like it only applied that property. This name implies the camera is shifted based on all of its changed properties.

Reply:
Good point, we have overused the word `target` here:

 - `private $targetSpherical` - the target destination of the camera
 - `public target` - the orbital focal point

Elsewhere, I have taken to referring to the first one as the "destination" of the  camera, so what would you say to `jumpToDestination`?

Reply:
Sounds good